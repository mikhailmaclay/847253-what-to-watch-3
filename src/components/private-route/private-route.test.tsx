// Libraries
import * as React from 'react';
import * as renderer from 'react-test-renderer';
import {Router} from 'react-router-dom';
// Components
import PrivateRoute from './private-route';
//
import history from '../../history';

describe(`<PrivateRoute/>`, () => {
  const props = {
    path: `/test`,
    exact: true,
    isAuthorized: true
  };

  it(`Should render correctly`, () => {
    history.push(`/test`);

    const result = renderer.create(
        <Router history={history}>
          <PrivateRoute {...props}>Test</PrivateRoute>
        </Router>
    ).toJSON();

    expect(result).toMatchSnapshot();
  });
});
