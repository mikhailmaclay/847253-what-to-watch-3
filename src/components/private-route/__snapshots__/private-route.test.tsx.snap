// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PrivateRoute/> Should render correctly 1`] = `"Test"`;
